sqlglot/dialects/doris.py

```diff
@@ -575,3 +657,23 @@ class Generator(MySQL.Generator):
             "xor",
             "year",
         }
+
+        def partition_sql(self, expression):
+            # Doris expects: PARTITION <name> VALUES LESS THAN (<value>)
+            # Each expression is a PartitionRange
+            parts = []
+            for e in expression.expressions:
+                parts.append(self.sql(e))
+            return ", ".join(parts)
```

Comment:
This doesn't look right, `Partition` is also constructed for statements like `INSERT OVERWRITE ...` ([docs](https://doris.apache.org/docs/2.0/sql-manual/sql-reference/Data-Manipulation-Statements/Manipulation/INSERT-OVERWRITE)). This override will omit the parentheses, which may result in invalid syntax.

I'd suggest only overriding the generation logic if you can guarantee that the `Partition` expression is used in a context where you need to use the unwrapped CSV syntax, e.g., do an instance check or something. Otherwise, fall back to the `super().partition_sql` implementation.

Also, you could condense what you have as follows:

```suggestion
return ", ".join(self.sql(e) for e in expression.expressions)
```